cleverpush/src/main/java/com/cleverpush/banner/models/blocks/BannerImageBlock.java

```diff
@@ -87,7 +87,9 @@ public static BannerImageBlock createImageBlock(JSONObject json) throws JSONExce
       }
     }
 
-    imageBlock.id = json.getString("id");
+    if (json.has("id")) {
+      imageBlock.id = json.getString("id");
```

Comment:
Can't it happen also for all other blocks / fields?

We can also use `optString` to prevent errors (also above for `scale`, `dismiss`, etc) it would make sense to use the `optXXX` methods)

@cpKevinrodriguesnobre  any idea why the block had no `id` set ? This should not happen anyways

Reply:
@gbhrdt 
I'll change the getXXX to optXXX for all the variables. and also change the other blocks as well.

Reply:
dont know maybe an edge case in frontent appBannerEditor

Reply:
I have updated the getXXX to optXXX